Error caught:
[Error: my error]
Finally
Done
